Sources/BraintreeCore/BTHTTP.swift

```diff
@@ -52,7 +52,7 @@ class BTHTTP: NSObject, URLSessionTaskDelegate {
     }
 
     var acceptLanguageString: String {
-        "\(Locale.current.language.languageCode?.identifier ?? "en")-\(Locale.current.region?.identifier ?? "US")"
+        "\(Locale.current.languageCode ?? "en")-\(Locale.current.regionCode ?? "US")"
```

Comment:
You might have to rebase, or merge in `v7`. Looks like a lot of old changes weren't in your branch